destination_sdk.proto

```diff
@@ -18,6 +19,29 @@ service Destination {
     rpc WriteBatch (WriteBatchRequest) returns (WriteBatchResponse) {}
 }
 
+message CapabilitiesRequest {}
+
+message CapabilitiesResponse {
+    repeated DataTypeMappingEntry data_type_mappings = 1;
+    bool supports_history_mode = 2;
+}
+
+message DataTypeMappingEntry {
+    DataType fivetran_type = 1;
+    oneof destination_type {
+        DestinationType map_to = 2;
+        bool unsupported = 3;
+    }
+}
+
+message DestinationType {
```

Comment:
This is missing the max value change that @fivetran-abdulsalam was working on. Is that coming in a separate PR?

Reply:
yes, that will be done in a separate PR

Reply:
which PR is that? can you link it here? Better yet, can we merge it into this one so I can see all the proto changes in one PR?

Reply:
That [ticket](https://fivetran.height.app/T-686139) is in progress. I've created a [subtask](https://fivetran.height.app/T-699973) under it to add max value support in examples. I'll pick the subtask once the parent PR is ready